presentation/src/main/java/com/knocklock/presentation/lockscreen/password/PassWordScreenStateHolder.kt

```diff
@@ -0,0 +1,78 @@
+package com.knocklock.presentation.lockscreen.password
+
+import android.content.res.Configuration
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.unit.dp
+
+/**
+ * @Created by 김현국 2023/01/11
+ * @Time 5:24 PM
+ */
+
+@Stable
+class PassWordScreenStateHolder(
+    configuration: Configuration,
+    private val removePassWordScreen: () -> Unit
+) {
+
+    val passWordSpace = 50.dp
+    val contentPadding = passWordSpace / 2
+    private val screenWidthDp = configuration.screenWidthDp.dp - passWordSpace * 3
+    val circlePassWordNumberSize = screenWidthDp / 3
+    val passWordState = mutableStateListOf(
+        PassWord(""),
+        PassWord(""),
+        PassWord(""),
+        PassWord(""),
+        PassWord(""),
+        PassWord("")
+    )
```

Comment:
```suggestion
  val passWordState = mutableStateListOf<PassWord>().apply {
        repeat(6) {
            add(PassWord(""))
        }
    }
```
repeat으로 좀 더 깔끔하게 할 수 있을거 같아용

Reply:
@KimHance 더깔끔하네영 반영할께용

Reply:
@KimHance 긴급 전화같은경우는 추후에 찾아볼께용

Reply:
@KimHance [e2b4719](https://github.com/Knock-Lock/Knock-Lock/pull/70/commits/e2b4719a8b976ef1e6432ad9459bb98cf72e0c9d)반영했습니다